specification/ai/HealthInsights/HealthInsights.Common/model.common.request.tsp

```diff
@@ -116,9 +116,10 @@ model PatientDocument {
   language?: string;
 
   @doc("The date and time when the document was created.")
-  createdDateTime?: utcDateTime;
+  createdAt?: utcDateTime;
 
   @doc("Document author(s)")
+  @maxItems(1)
```

Comment:
Why is there a maxItem for 1 if there can be multiple authors in the array?

Reply:
Did you mean minItems?

Reply:
Good catch. Thank you!